release_images.yml

```diff
@@ -181,4 +181,53 @@ release_images:
       example: False
       disable_sm_tag: False
       force_release: False
-  
\ No newline at end of file
+  13:
+    framework: "pytorch"
+    version: "1.12.0"
+    arch_type: "x86"
+    customer_type: "e3"
+    training:
+      device_types: ["cpu", "gpu" ]
+      python_versions: [ "py38" ]
+      os_version: "ubuntu20.04"
+      cuda_version: "cu116"
+      example: False         # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+      # to images being published.
+      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
+      # has already been published. Re-released image will have minor version incremented by 1.
+    inference:
+      device_types: [ "cpu", "gpu" ]
+      python_versions: [ "py38" ]
+      os_version: "ubuntu20.04"
+      cuda_version: "cu116"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+      # to images being published.
+      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
+      # has already been published. Re-released image will have minor version incremented by 1.
+  14:
+    framework: "pytorch"
+    version: "1.12.0"
+    arch_type: "x86"
+    customer_type: "sagemaker"
```

Comment:
I don't see any dockerfiles or image specifications for PT 1.12 SM DLCs. Is this release_images.yml entry valid?

Reply:
I put it here for SM release convenience. There is no SM RC been build, and I disabled the SM release pipeline to prevent unexpected release.

Reply:
removed SM entry